foundation/foundation-configuration-store/src/test/resources/specifications/load-configurations.feature

```diff
@@ -69,3 +69,9 @@ Feature: Load configurations at specified URI based on environment context
     Given there exists a krausening_password and encrypted properties
     When the configuration service starts
     Then the loaded properties contains the decrypted value
+
+  @config-webhook
+  Scenario: The configuration service can process Kubernetes resources
```

Comment:
A: I'd consider moving this to it's own feature file since we know we'll have multiple scenarios related to the webhook.

Reply:
Moved to it's own feature file.